src/pages/lesson-plans/lesson-06.html

```diff
@@ -447,123 +455,143 @@ <h2 class="main">Integrated Lesson # 6</h2>
 <p>[Students respond. On the Velcro board next to the goal.]</p>
-</step></li><li><step><p>“This will remind us of what CAUSED the character to have the goal. We place this Because Card on the Velcro board,</p>
+</step></li>
+<li><step [stepsContext]="steps" [highlightedStepId]="suggestedStepId" [currentQuestionStepId]="gettingResponsesFor"><p>“This will remind us of what CAUSED the character to have the goal. We place this Because Card on the Velcro board,</p>
 <p>[hold up the blue Because Card]</p>
 <p> and we put it right up here next to the goal.”</p>
 <p>[Place the blue Because card to the LEFT of the thought bubble.]</p>
-</step></li><li><step><p>“Since this is a CAUSE, we can connect the blue Because Card to what it CAUSED. So, we can use a Because String to connect the blue Because Card to the goal.”</p>
+</step></li>
+<li><step [stepsContext]="steps" [highlightedStepId]="suggestedStepId" [currentQuestionStepId]="gettingResponsesFor"><p>“Since this is a CAUSE, we can connect the blue Because Card to what it CAUSED. So, we can use a Because String to connect the blue Because Card to the goal.”</p>
 <p>[Point to wikki stick connecting the Because Card to a thought bubble.]</p>
-</step></li><li><step><p>We can retell the story by saying, “I want to...” and tell what the goal is, and then say <em>“because”</em> and know WHY the character wanted to do that..</p>
+</step></li>
+<li><step [stepsContext]="steps" [highlightedStepId]="suggestedStepId" [currentQuestionStepId]="gettingResponsesFor"><p>We can retell the story by saying, “I want to...” and tell what the goal is, and then say <em>“because”</em> and know WHY the character wanted to do that..</p>
 <p>[Trace finger along the Because String while saying because]</p>
 </step></li></ul>
 <p class="v-padding bullet-heading"><strong>Next, review finding TRIES: </strong></p>
-<ul><li><step><p>Ask the students, “Once we know what a character’s goal is, what do we look for next?”<br />
-[What a character did to try to get their goal.]</p></step></li><li><step>
+<ul><li><step [stepsContext]="steps" [highlightedStepId]="suggestedStepId" [currentQuestionStepId]="gettingResponsesFor"><p>Ask the students, “Once we know what a character’s goal is, what do we look for next?”<br />
+[What a character did to try to get their goal.]</p></step></li>
+<li><step [stepsContext]="steps" [highlightedStepId]="suggestedStepId" [currentQuestionStepId]="gettingResponsesFor">
 <p>Ask: “What is a try?”</p>
 
 
 <p>[Tries are actions/things a character does to try to get what they want/the goal.]</p>
-</step></li><li><step>
+</step></li>
+<li><step questionType="goal" correctAnswer="try" (onGetResponses)="getResponses($event)" [stepsContext]="steps" [highlightedStepId]="suggestedStepId" [currentQuestionStepId]="gettingResponsesFor">
```

Comment:
here's an example of a question step -- it has all the same things as a regular step, plus it registers the `(onGetResponses)="getResponses($event)"` listener, and it also has question type and (in this case) the correct answer